changelog.yaml

```diff
@@ -1034,3 +1034,10 @@
     fixed:
     - Fixed the CTC formula in 2021 (only) to correctly apply the income-based reduction.
   date: 2022-07-03 21:47:43
+- bump: minor
+  changes:
+    added:
+    - MO Income Tax.
+    - MO Income Tax before credits.
+    - Parameters for MO Tax Schedule.
+  date: 2022-07-04 17:08:31
```

Comment:
We actually don't need to add this line, it gets added automatically with the time stamp of the merge.